theme/scripts/cart.ts

```diff
@@ -284,27 +280,31 @@ export default(function() {
         }
       })
     },
+  
+    handleProductAddedToCart() {
+      const desktopProductAddToCartButton = document.querySelector(Cart.SELECTORS.desktopProductAddToCartButton);
+      const mobileProductAddToCartButton = document.querySelector(Cart.SELECTORS.mobileProductAddToCartButton);
 
-    //TO-DO (#178): This function should check when the add to cart button is clicked and use a setTimeout to change the text to "Added to cart" and then revert back to "Add to cart" after a few seconds.
-    handleAddToCartButton(variantId: string, action: Action) {
-      const addToCartButton = document.querySelector(Cart.SELECTORS.addToCartButton);
-      
-      if (!addToCartButton) return;
-      
-      const addToCartVariantId = addToCartButton.getAttribute(Cart.ATTRIBUTES.variantId);
-
-      if (addToCartVariantId === variantId) {
-        if (action === Action.ADD) {
-          addToCartButton.textContent = Cart.CONSTANTS.addedToCart;
-          addToCartButton.setAttribute(Cart.ATTRIBUTES.disabled, Cart.ATTRIBUTES.disabled);
-        }
+      if (desktopProductAddToCartButton) {
+        console.log('button is here desktop')
```

Comment:
We can probably remove these `console.log`s

Reply:
Hahahha embarrassing! And yes this is the reason why we weren't seeing the buttons change... so silly!!